### ISSUE TITLE
The repeat loop suggestion

### ISSUE THREAD

**Issue by idealkindom** [Proektsoftbg/Calcpad#102]
How about adding repeat loop syntax like "do while " and "unless do".

**Comment by Proektsoftbg** [comment]
Yes, this is quite possible. Just a syntax sugar :). I think that "until" is more used in programming languages than "unless" for that purpose.

**Comment by idealkindom** [comment]
> Yes, this is quite possible. Just a syntax sugar :). I think that "until" is more used in programming languages than "unless" for that purpose.

Yes, maybe only Maxima use "unless do".
